prod/westeurope/eucovidcert/functions_eucovidcert/function_app/terragrunt.hcl

```diff
@@ -105,7 +105,8 @@ inputs = {
     FETCH_KEEPALIVE_TIMEOUT             = "60000"
 
     DGC_UAT_FISCAL_CODES   = local.testusersvars.locals.test_users_eu_covid_cert_flat
-    LOAD_TEST_FISCAL_CODES = local.testusersvars.locals.test_users_internal_load_flat
+    # we need test_users_store_review_flat because app IO reviewers must read a valid certificate response
+    LOAD_TEST_FISCAL_CODES = "${local.testusersvars.locals.test_users_store_review_flat},${local.testusersvars.locals.test_users_internal_load_flat}"
```

Comment:
```suggestion
    LOAD_TEST_FISCAL_CODES = join("," [local.testusersvars.locals.test_users_store_review_flat, local.testusersvars.locals.test_users_internal_load_flat])
```

Reply:
Function first :-) .... not mandatory.

Reply:
thanks, done in https://github.com/pagopa/io-infrastructure-live-new/pull/551/commits/5f666b289c9f122d0b65d101817030d16ea4f288